share/power-bi/README.md

```diff
@@ -0,0 +1,82 @@
+To connect Power BI to the Uni Node, you must obtain the ***GraphQL URL*** and ***GraphQL API Key*** values of a Node in your Uni. You can access these values by visiting the [Uni Dashboard](https://share.vendia.net/)
+
+### Step 2 - Create a Power BI Query
+Power BI does not have a native GraphQL connector. As such, you must create a [Blank Query](https://docs.microsoft.com/en-us/power-bi/transform-model/desktop-query-overview).  You can do so by clicking the ***Get data*** toolbar icon from the Home toolbar menu as shown
+
+![img/create_query.png](img/create_query.png)
+
+With the Power Query Editor window open, you can rename the query by right-clicking on it in the queries toolbar (left) and selecting rename. For the purposes of this example, the first query will be named InventoryItems as shown
+
+![img/rename_query.png](img/rename_query.png)
+
+Switch to the [Advanced Editor View](https://docs.microsoft.com/en-us/power-bi/transform-model/desktop-query-overview#advanced-editor) by clicking the View menu and selecting the Advanced Editor toolbar button.
+
+Replace the query text using the [example_inventory_item_query Example](example_inventory_item_query.pqfl) as a template. This template defines a query for InventoryItems that are defined in the [Example Schema](example_schema.json) as shown
+
+> **Note** You must replace the sample GraphQL URL and API Key with those from your Uni Node. These values are retrievable from the [Uni Dashboard](https://share.vendia.net)
+
+![img/advanced_inventory_items.png](img/advanced_inventory_items.png)
+
+If prompted by a message indicating "Please Specify How To Connect" press the Edit Credentials button and choose Anonymous (which should be the default setting). Anonymous authentication is used as the x-api-key header in the script carries the identification for the request.
+
+![img/edit_credentials.png](img/edit_credentials.png)
+
+If the request worked successfully, a data table should be displayed
+
+![img/items.png](img/items.png)
+
+To complete the query setup save the Query and the Workbook
+
+### Step 3 - Add a control
+
+With the query saved, you can start visualizing the data. Try adding a Table control from the Visualizations Panel. Populate it by selecting it and clicking the "Value" property of InventoryItems in the Fields Panel as shown
+
+![img/data_table.png](img/data_table.png)
+
+### Step 4 - Transforms
+
+Sometimes data returned from List operations is not formatted such that it can be visualized for all controls. Transforms allow for manipulation of the data after it is retrieved. Add a new transform by clicking the Data Transformation toolbar item from the Home menu.
+
+Click the "Count Rows" transformation in the query editor to easily transform the list of Inventory Items into an aggregate Count of rows.
+
+Save the query and re-visit the table.
+
+Update the table by selecting the new aggregate field and clicking the "fix this" button on the table.
```

Comment:
Worth having a visualization of the process?  It might be helpful to show the expected result given the dataset.